Facepunch.Steamworks/Generated/SteamEnums.cs

```diff
@@ -752,7 +752,7 @@ public enum GamepadTextInputLineMode : int
 	//
 	// ETextFilteringContext
 	//
-	internal enum TextFilteringContext : int
+	public enum TextFilteringContext : int
```

Comment:
Don't change anything in generated folder

Reply:
Generated code is now what the generator should give as a result